test/hcal_digi_pipeline_test_config.py

```diff
@@ -0,0 +1,36 @@
+
+from LDMX.Framework import ldmxcfg
+
+# Create a process
+p = ldmxcfg.Process( 'test_hcal_digis' )
+p.libraries.append("libHcal.so")
+
+# Set the maximum number of events
+p.maxEvents = 10 # should be the same as NUM_TEST_SIM_HITS
+
+# Import the Hcal conditions 
+from LDMX.Hcal import digi
+
+# Set the output file name
+p.outputFiles = ['hcal_digi_pipeline_test.root']
+
+# The the histogram file name
+p.histogramFile = 'hcal_digi_pipeline_test_histo.root'
+
+# Geometry provider
+from LDMX.Hcal import HcalGeometry
+geom = HcalGeometry.HcalGeometryProvider.getInstance()
```

Comment:
This line also shouldn't be needed. It is nice for the end user to just
```python
import LDMX.Hcal.HcalGeometry
```
and then be done! (Kind of reads well).